README.md

```diff
@@ -17,6 +17,10 @@ Each org builds its own Hugo site that uses the common theme and content modules
 - [CodeYourFuture](/org-cyf/) => [https://org-cyf-theme.netlify.app/](https://org-cyf-theme.netlify.app/)
 - [MigraCode](/org-mcb/) => [https://org-mcb-theme.netlify.app/](https://org-mcb-theme.netlify.app/) (couldn't find an svg logo)
```

Comment:
```suggestion

```
They moved to https://github.com/Migracode-Barcelona/curriculum

Reply:
Thanks, removed.